README.md

```diff
@@ -1,57 +1,52 @@
-# Project Name
+# azure-data-tables sample application
 
-(short, 1-3 sentenced, description of the project)
+This is the sample project for using the [azure-data-tables SDK](https://search.maven.org/artifact/com.azure/azure-data-tables) library with the Azure Cosmos DB Table API.  The quickstart for this application is located at [https://docs.microsoft.com/azure/cosmos-db/table/create-table-java](https://docs.microsoft.com/azure/cosmos-db/table/create-table-java).
```

Comment:
```suggestion
This is the sample project for using the [azure-data-tables client library](https://search.maven.org/artifact/com.azure/azure-data-tables) with the Azure Cosmos DB Table API. The quickstart for this application is located at [https://docs.microsoft.com/azure/cosmos-db/table/create-table-java](https://docs.microsoft.com/azure/cosmos-db/table/create-table-java).
```

Reply:
Should we point to Maven instead to our own Microsoft Docs page or GitHub repository @mario-guerra?